### PR TITLE
Features: Detail

### PR DESCRIPTION
**Pull request by jshaughn**
Part-of https://github.com/kiali/kiali/issues/4118

Update the Detail feature section, and migrate custom dash config from the feature section to its own section in Configuration.

### CODE REVIEW

File: content/en/docs/Features/details.md [hunk 1]
```diff
@@ -1,210 +1,108 @@
...
-      </a>
-    </span>
-</div>
+![Detail list apps](/images/documentation/features/detail-list-app.png)
+![Detail list service](/images/documentation/features/detail-list-service.png)
+![Detail list workload](/images/documentation/features/detail-list-workload.png)
+![Detail list Istio config](/images/documentation/features/detail-list-config.png)
 
-## Overview Tab
+</br>
```
Comment (israel-hdez) [inline content/en/docs/Features/details.md]: I saw several `</br>` tags.
You probably wanted to use `<br />` (slash at the end). See: https://dev.w3.org/html5/html-author/#tags (search for "self-closing tags" text)

File: content/en/docs/Configuration/custom-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+  externalLinks:
+  - name: "My custom Grafana dashboard"
+    type: "grafana"
+    variables:
+      app: var-app
+      namespace: var-namespace
+      version: var-version
+```
+
+The *name* field corresponds to what you can set in the pod annotation #pods-annotations[`kiali.io/dashboards`].
```
Comment (israel-hdez) [inline content/en/docs/Configuration/custom-dashboard.md]: This anchor is not being converted to a link :-/

File: content/en/docs/Configuration/custom-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+  * *include*: to include another dashboard, or a specific chart from another dashboard. Typically used to compose with generic dashboards such as the ones about _MicroProfile Metrics_ or _Micrometer_-based JVM metrics. To reference a full dashboard, set the name of that dashboard. To reference a specific chart of another dashboard, set the name of the dashboard followed by `$` and the name of the chart (ex: `include: "microprofile-1.1$Thread count"`).
+* *externalLinks*: a list of related external links (e.g. to Grafana dashboards)
+  * *name*: name of the related dashboard in the external system (e.g. name of a Grafana dashboard)
+  * *type*: link type, currently only _grafana_ is allowed
+  * *variables*: a set of variables that can be injected in the URL. For instance, with something like _namespace: var-namespace_ and _app: var-app_, an URL to a Grafana dashboard that manages _namespace_ and _app_ variables would look like:
+_\http://grafana-server:3000/d/xyz/my-grafana-dashboard?var-namespace=some-namespace&var-app=some-app_. The available variables in this context are *namespace*, *app* and *version*.
```
Comment (israel-hdez) [inline content/en/docs/Configuration/custom-dashboard.md]: The backslash in the grafana url seems to be specific for adoc. It was to escape the url and prevent asciidoc to render it as a link. But markdown is rendering the backslash. Probably it is not needed.

File: content/en/docs/Features/details.md [hunk 4]
```diff
@@ -1,210 +1,108 @@
...
-    <span style="margin: 0 auto;">
-      <a class="image-popup-fit-height" href="/images/documentation/features/service-list-v1.22.0.png" title="List Services">
-          <img src="/images/documentation/features/service-list-thumb-v1.22.0.png" style="width: 660px; display:inline;margin: 0 auto;" />
-      </a>
-      <a class="image-popup-fit-height" href="/images/documentation/features/istio-list-v1.22.0.png" title="List Istio Config">
-          <img src="/images/documentation/features/istio-list-thumb-v1.22.0.png" style="width: 660px; display:inline;margin: 0 auto;" />
-      </a>
-    </span>
-</div>
+![Detail list apps](/images/documentation/features/detail-list-app.png)
```
Comment (israel-hdez) [inline content/en/docs/Features/details.md:14]: :+1: Good simplification.
Thanks for doing this work.

File: content/en/docs/Features/details.md [hunk 5]
```diff
@@ -1,210 +1,108 @@
...
-In the Workload Overview, Kiali performs several validations on workload configuration:
+Each Overview provides:
 
-* Are Istio sidecars deployed?
-* Are proper *app* and *version* labels assigned?
+* links to related components and linked Istio configuration.
+* health status.
+* validation information.
+* an Action menu for actions that can be taken on the component.
+  * several [Wizards](#wizards) are available.
```
Comment (israel-hdez) [inline content/en/docs/Features/details.md]: Link is broken. No Wizards section yet?

**Review by israel-hdez — CHANGES_REQUESTED** [review]
You are doing an awesome job on updating and expanding feature descriptions.

I have minor comments.

File: content/en/docs/Features/details.md (hunk 1 above)
Comment (jshaughn) [reply content/en/docs/Features/details.md]: ok, I'll change those, thanks.

File: content/en/docs/Configuration/custom-dashboard.md (hunk 2 above)
Comment (jshaughn) [reply content/en/docs/Configuration/custom-dashboard.md]: fixed.

File: content/en/docs/Configuration/custom-dashboard.md (hunk 3 above)
Comment (jshaughn) [reply content/en/docs/Configuration/custom-dashboard.md]: fixed.

File: content/en/docs/Features/details.md (hunk 5 above)
Comment (jshaughn) [reply content/en/docs/Features/details.md]: Fixed.

**Comment by jshaughn** [comment]
@israel-hdez , thanks for the feedback, should be fixed.  In a separate PR I'll clean up the remaining </br> and also go back and consolidate the use of the markdown format for images.  In the earlier work, like for the tutorial, I had left the embedded html.  But I think it's better to consolidate so that any changes we may make to styling are assure of being applied consistently.

**Review by israel-hdez — APPROVED** [review]
I still see some `</br>` tags.

It's OK since you will be fixing in a follow up PR. Also, Chrome seems to somehow fallback and render correctly.

LG!

**Pull request merged**